app/code/Magento/Customer/Model/Metadata/Form/Text.php

```diff
@@ -126,7 +124,7 @@ private function validateLength($value, AttributeMetadataInterface $attribute, a
         // validate length
         $label = __($attribute->getStoreLabel());
 
-        $length = $this->_string->strlen(trim($value));
+        $length = $this->_string->strlen(trim((string) $value));
```

Comment:
Hello @SilinMykola 
What do you think about suggestion proposed below?

```suggestion
        $length = is_string($value) ? $this->_string->strlen(trim($value)) : 0;
```

Reply:
Hello @andrewbess
Thanks for your review. I made changes, please check.